engine/lib/users.php

```diff
@@ -481,6 +481,11 @@ function elgg_user_hover_menu($hook, $type, $return, $params) {
 		$item = new \ElggMenuItem('avatar:edit', elgg_echo('avatar:edit'), $url);
 		$item->setSection('action');
 		$return[] = $item;
+
+		$url = "settings/user/$user->username";
+		$item = new \ElggMenuItem('settings:edit', elgg_echo('settings:edit'), $url);
+		$item->setSection('action');
+		$return[] = $item;
```

Comment:
Only if viewing other admin's hover menu. I would say let's add $user->canEdit() section, and move all the stuff there, except for truly admin stuff.